spec/factories/storage_locations.rb

```diff
@@ -59,5 +59,38 @@
         end
       end
     end
+
+    trait :with_items_mixed do
```

Comment:
This is only used in one place and is pretty complex. I'd much rather just have that one spec create this data rather than adding it to the factory.

Reply:
Sure thing.  See [6d00cbc](https://github.com/rubyforgood/human-essentials/pull/4804/commits/6d00cbc43dd8c544400c2988404801ad2dc7fc4d).